tools/docs/config.ts

```diff
@@ -76,7 +76,7 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
     .filter((option) => option.releaseStatus !== 'unpublished')
     .forEach((option) => {
       // TODO: fix types (#9610)
-      const el: Record<string, any> = { ...option };
+      const el: Record<string, any> = { mergeable: false, ...option };
```

Comment:
I think mergeable only makes sense for arrays and objects so may confuse people if in the docs for strings and numbers